docs/source/api/schedulers.mdx

```diff
@@ -56,128 +56,3 @@ The class [`SchedulerOutput`] contains the outputs from any schedulers `step(...
 
 [[autodoc]] schedulers.scheduling_utils.SchedulerOutput
```

Comment:
Let's maybe fully remove this doc page and move everything which is still left here to "overview.mdx"

Reply:
> Also, maybe we can add a brief discussion about schedulers working with different objectives (epsilon, sample, or v-prediction), but that can come in another PR.

good idea! I can add this to the overview page